v2/account_claims.go

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2018-2019 The NATS Authors
```

Comment:
Date... unless those files were moved, it should start with the year the file was first created.